src/display/annotation_layer.js

```diff
@@ -197,6 +201,52 @@ class AnnotationElement {
     return AnnotationElement._hasPopupData(this.data);
   }
 
+  updateEdited(params) {
+    if (!this.container) {
+      return;
+    }
+
+    this.#updates ||= {
+      rect: this.data.rect.slice(0),
+    };
+
+    const { rect } = params;
+
+    if (rect) {
+      this.#setRect(rect);
+    }
+  }
+
+  resetEdited() {
+    if (!this.#updates) {
+      return;
+    }
+    this.#setRect(this.#updates.rect);
+    this.#updates = null;
+  }
+
+  #setRect(rect) {
```

Comment:
Maybe update this name as well, to avoid accidental future use elsewhere?
```suggestion
  #setEditedRect(rect) {
```